src/styles/index.scss

```diff
@@ -1,3 +1,6 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+
 body {
   margin: 0;
+  font-family: roboto, sans-serif;
```

Comment:
Font family names should be quoted if they are not a single word. Consider using 'Roboto', sans-serif.